src/views/OurTeamPageView/TeamMemberCard/TeamMemberCard.styles.js

```diff
@@ -23,6 +37,32 @@ export const galleryStyleCard = css`
 export const MemberCard = styled.div`
   position: relative;
 
+  @media screen and (prefers-reduced-motion: reduce) {
```

Comment:
Il n'est pas nécessaire d'appliquer `@media screen and (prefers-reduced-motion: reduce)` sur tout le pseudo element mais juste sur la propriété `transition`. De plus, ça duplique du code pour rien.
```suggestion
  &::after {
    position: absolute;
    top: 0;

    display: block;
    width: 100%;
    height: 100%;

    background: linear-gradient(
      180deg,
      rgba(0, 0, 0, 0) 0%,
      rgba(0, 0, 0, 0) 60%,
      rgba(30, 30, 80, 0.8) 100%
    );

    border-radius: 12px;

    opacity: 0;

    @media screen and (prefers-reduced-motion: no-preference) {
      transition: opacity ${speed.fast};
    }

    content: '';
  }
```